src/commands/sites/sites-create-template.js

```diff
@@ -116,6 +117,15 @@ const sitesCreateTemplate = async (options, command) => {
             name: siteName,
           },
         })
+        const { cloneConfirm } = await inquirer.prompt({
+          type: 'confirm',
+          name: 'cloneConfirm',
+          message: `Do you want to clone the repository? You will find it in the git-clones folder`,
+          default: false,
+        })
+        if (cloneConfirm) {
+          clone(repoResp.html_url, `git-clones/${repoResp.name}`)
```

Comment:
1. We should use the `promise` version or wait on the callback for completion, https://github.com/jaz303/git-clone/tree/1f00fa820a2f22abff2fda7ed60c6bec7e24d70c#promise.

2. Might be better to use the `git` URL for cloning as that's more common

3. Can we drop the `git-clones`? `git clone` should a directory for the repo, correct?

Reply:
in which folder should we save the cloned repo?

Reply:
>in which folder should we save the cloned repo?

`git clone` should create a folder in the current working directory. I think that's the desired behavior, WDYT?

Reply:
Ok. I will change it